examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -569,20 +568,43 @@ function vertical_diffusion_boundary_layer_cache(
 
     cond_type = NamedTuple{(:shf, :lhf, :E, :ρτxz, :ρτyz), NTuple{5, FT}}
 
-    return (;
-        ᶠv_a = similar(Y.f, eltype(Y.c.uₕ)),
-        ᶠz_a,
-        ᶠK_E = similar(Y.f, FT),
-        surface_conditions = similar(z_bottom, cond_type),
-        dif_flux_uₕ,
-        dif_flux_energy = similar(z_bottom, Geometry.WVector{FT}),
-        dif_flux_ρq_tot,
-        Cd,
-        Ch,
-        diffuse_momentum,
-        coupled,
-        z_bottom,
-    )
+    if surface_scheme == "bulk"
+        Cd = FT(0.0044)
+        Ch = FT(0.0044)
+        return (;
+            surface_scheme,
+            ᶠv_a = similar(Y.f, eltype(Y.c.uₕ)),
+            ᶠz_a,
+            ᶠK_E = similar(Y.f, FT),
+            surface_conditions = similar(z_bottom, cond_type),
+            dif_flux_uₕ,
+            dif_flux_energy = similar(z_bottom, Geometry.WVector{FT}),
+            dif_flux_ρq_tot,
+            Cd,
+            Ch,
+            diffuse_momentum,
+            coupled,
+            z_bottom,
+        )
+    elseif surface_scheme == "monin_obukhov"
+        z0m = FT(1e-5)
+        z0b = FT(1e-5)
+        return (;
+            surface_scheme,
+            ᶠv_a = similar(Y.f, eltype(Y.c.uₕ)),
+            ᶠz_a,
+            ᶠK_E = similar(Y.f, FT),
+            surface_conditions = similar(z_bottom, cond_type),
+            dif_flux_uₕ,
+            dif_flux_energy = similar(z_bottom, Geometry.WVector{FT}),
+            dif_flux_ρq_tot,
+            z0m,
+            z0b,
+            diffuse_momentum,
+            coupled,
+            z_bottom,
+        )
+    end
```

Comment:
```suggestion
    surface_scheme_kwargs = if surface_scheme == "bulk"
        (; Cd = FT(0.0044), Ch = FT(0.0044))
    elseif surface_scheme == "monin_obukhov"
        (; z0m = FT(1e-5), z0b = FT(1e-5))
    end
    return (;
        surface_scheme,
        ᶠv_a = similar(Y.f, eltype(Y.c.uₕ)),
        ᶠz_a,
        ᶠK_E = similar(Y.f, FT),
        surface_conditions = similar(z_bottom, cond_type),
        dif_flux_uₕ,
        dif_flux_energy = similar(z_bottom, Geometry.WVector{FT}),
        dif_flux_ρq_tot,
        surface_scheme_kwargs...,
        diffuse_momentum,
        coupled,
        z_bottom,
    )
```
If you're going to fix the label, then we might as well write this more compactly 🙂

Reply:
We can probably do the same below, too.

Reply:
Let’s fix it. Should we revert the PR or just open a new PR?